DevAcademy/Model/Feature.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  Feature.swift
+//  DevAcademy
+//
+//  Created by Michal Wagner on 25.07.2023.
+//
+
+import Foundation
```

Comment:
Vypada to, ze tady chybi ty samotne `structy`, ktere se stale nachazi v souboru `DataModel` ...mrkni jestli jsi pushnul vsechny commity. 🕵️‍♂️ 😉